_raml/apps/orders/examples/get_orders_response.json

```diff
@@ -37,11 +37,34 @@
       "grandTotal": "250.94",
       "totalBeforeTax": "250.94",
       "totalTax": "0",
-      "comment": null,
+      "customerComment": null,
+      "internalNote": null,
       "cancelledOn": null,
       "closedOn": null,
-      "paidOn": null,
+      "paidOn": "2016-01-20T07:23:47.000Z",
       "returnedOn": null,
+      "shippingData": {
+        "shippingMethod": {
+          "id": "5171DD86-C077-0726-C448-AC14145F9026",
+          "name": "Standard delivery"
+        },
+        "price": {
+          "taxType": "GROSS",
+          "formatted": "7.63 €",
+          "amount": 7.63,
+          "currency": "EUR"
+        }
+      },
+      "paymentData": {
+        "paymentMethod": {
+          "id": "5171DD86-0714-8E9B-B079-AC14145F900F",
+          "name": "Payment in advance"
+        },
+        "price": {
+          "taxType": "GROSS",
+          "formatted": "250.94 €",
+          "amount": 250.94,
+          "currency": "EUR"
       "links": [
```

Comment:
add link to shipping method if there is "shippingData":

{
          "rel": "shipping-method",
          "href": "https://pm.epages.com/rs/shops/apidocu/shipping-methods/5171DD86-C077-0726-C448-AC14145F9026"
        }